test cases/objc/6 objfw with modules/meson.build

```diff
@@ -0,0 +1,9 @@
+project('SimpleTest', 'objc')
+
+objfw_dep = dependency('objfw', modules: ['ObjFWTest'], required: false)
+if not objfw_dep.found()
+  error('MESON_SKIP_TEST: Need objfw dependency')
+endif
+
+executable('SimpleTest', 'SimpleTest.m',
+  dependencies: [objfw_dep])
```

Comment:
Please merge the new 5 and 6 tests into a single test. Meson startup overhead is the longest pull in CI, and so we've moved to have fewer, but more complex tests rather than many simple ones.

Reply:
ObjFWTest is very new it looks like, which means that it's very likely for people with the framework installed to not have this and end up with the whole test being skipped.

Is it possible to use a different module that has been around for longer to do the test that `modules:` works?

Reply:
@dcbaker Done.
@eli-schwartz Unfortunately, ObjFWTest is the first module ObjFW ships with. All others are 3rd party.